frontend/src/components/clinics/components/common/constants/default-clinic-value.constant.ts

```diff
@@ -7,6 +7,7 @@ const DEFAULT_CLINIC_VALUE: IClinicPayload = {
   [ClinicKey.ADDRESS]: '',
   [ClinicKey.IMAGE_PATH]: ClinicDefaultImage,
   [ClinicKey.CLINIC_TYPE]: ClinicType.STATE,
+  [ClinicKey.CITY_ID]:' ',
```

Comment:
Just make it nullable
```suggestion
  [ClinicKey.CITY_ID]: null,
```